src/example-relay/src/Homepage/locations/LocationsPaginated.js

```diff
@@ -1,71 +1,60 @@
 // @flow
 
-import {
-  createPaginationContainer,
-  graphql,
-  type PaginationRelayProp,
-  type PaginationContainerType,
-} from '@adeira/relay';
+import { type Node, useCallback } from 'react';
+import { graphql, usePaginationFragment } from '@adeira/relay';
 
 import Button from '../../components/Button';
 import Location from './Location';
 import LocationList from './LocationsList';
-import type { LocationsPaginated_data as LocationsDataType } from './__generated__/LocationsPaginated_data.graphql';
+import { type LocationsPaginatedRefetchQuery } from './__generated__/LocationsPaginatedRefetchQuery.graphql';
+import { type LocationsPaginated_data$key as LocationsFragmentRef } from './__generated__/LocationsPaginated_data.graphql';
```

Comment:
There is a cool way how to avoid aliasing the import (I assume because of `camelcase` lint rule?), _if you want_. Simply name the fragment the way you want it to be (without `*_data` postfix) and you are good to go. Using `<FileName>_<propName>` is an old convention that is no longer needed.